operations/mimir-mixin-compiled/dashboards/mimir-compactor.json

```
@@ -203,6 +203,455 @@
                         "show": false
                      }
                   ]
+               },
+               {
+                  "aliasColors": { },
+                  "bars": false,
+                  "dashLength": 10,
+                  "dashes": false,
+                  "datasource": "$datasource",
+                  "description": "### Longest time since last successful run\nDisplays the amount of time since the most recent successful execution\nof the compactor.\nThe value shown will be for the compactor replica which has the longest time since its\nlast successful run.\nThe table to the right shows a summary for all compactor replicas.\n\nIf there is no time value, one of the following messages may appear:\n\n- If you see \"No compactor data\" in this panel, that means that no compactors are active yet.\n\n- If you see \"No successful runs\" in this panel, that means that compactors are active, but none \n  of them were successfully executed yet.\n\nThese may be expected - for example, if you just recently restarted your compactors, they may not \nyet be broadcasting metrics, or they may not have had a chance to complete their first compaction run. \nHowever, if these messages persist, you should check the health of your compactors.\n\n",
```

```suggestion
                  "description": "### Longest time since last successful run\nDisplays the amount of time since the most recent successful execution\nof the compactor.\nThe value shown will be for the compactor replica that has the longest time since its\nlast successful run.\nThe table to the right shows a summary for all compactor replicas.\n\nIf there is no time value, one of the following messages might appear:\n\n- If you see \"No compactor data\" in this panel, that means that no compactors are active yet.\n\n- If you see \"No successful runs\" in this panel, that means that compactors are active, but none \n  of them were successfully executed yet.\n\nThese might be expected - for example, if you just recently restarted your compactors, they might not \nyet be broadcasting metrics, or they might not have had a chance to complete their first compaction run. \nHowever, if these messages persist, check the health of your compactors.\n\n",
```
I changed "may" to "might" to connote possibility instead of permission.